Arduboy.cpp

```diff
@@ -682,14 +687,12 @@ unsigned char* Arduboy::getBuffer()
 
 boolean Arduboy::pressed(uint8_t buttons)
 {
- uint8_t button_state = getInput();
- return (button_state & buttons) == buttons;
```

Comment:
The change doesn't seem to affect compile size.

Personally, I find my version clearer. I tend to avoid defining variables that are only used once, unless they're being assigned by some long, complicated expression. But, we can change it back, unless other people have an opinion.